src/components/BPageNav/BPageNav.test.js

```diff
@@ -0,0 +1,38 @@
+import { shallow } from 'vue-test-utils'
+import BPageNav from './BPageNav'
+import events from '../../event-bus'
+
+describe('Page nav', () => {
+  it('mounted html should match snapshot', () => {
+    const wrapper = shallow(BPageNav, {
+      propsData: {
+        links: [{ label: 'Link', href: 'link-1', completed: true }]
+      }
+    })
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+})
+
+describe('Page nav multiple links', () => {
+  it('mounted html should match snapshot', () => {
+    const wrapper = shallow(BPageNav, {
+      propsData: {
+        links: [{ label: 'Link', href: 'link-1', completed: true }, { label: 'Link2', href: 'link-2', completed: false }]
+      }
+    })
+    expect(wrapper.html()).toMatchSnapshot()
+  })
+})
+
+describe('Page nav', () => {
+  it('fire optional event on click', () => {
```

Comment:
Instead of nesting this under another `describe` with the same value as the first, could this be nested under the first `describe`?